app/assets/src/components/views/projects/projects_view.scss

```diff
@@ -16,15 +16,20 @@
   .projectDescription {
     color: $dark-grey;
     font-size: $font-size-h6;
-    margin-bottom: 8px;
     line-height: 16px;
-    height: 32px;
+    // remove description for now:
+    height: 0;
+    overflow: hidden;
+    // height: 32px;
```

Comment:
commented out because...? please add a comment in the code

Reply:
Comment above, temporary removal.
Just while descriptions do not make it to the database. I can add that to the comment.